app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb

```diff
@@ -54,7 +54,10 @@
             </div>
 
             <div class="govuk-radios__item">
-              <%= f.radio_button :temp_payment_method, "bank_transfer", class: "govuk-radios__input" %>
+              <%= f.radio_button :temp_payment_method,
+                  "bank_transfer",
+                  class: "govuk-radios__input",
+                  disabled: WasteCarriersEngine.configuration.host_is_back_office? %>
```

Comment:
Is the `host_is_back_office?` check necessary, as this view is only for back-office use?

Reply:
This is only to be disabled when used in the back office. I'm not sure this page ever gets hit in the front office, but it's in the engine so it could